### DISCUSSION TITLE
speedy-ts-compiler was removed......

### DISCUSSION THREAD

**Discussion by uk-taniyama** [TypeStrong/ts-node#1787] [Q&A] [answered]
Security holding package
This package contained malicious code and was removed from the registry by the npm security team. A placeholder was published to ensure users are not affected in the future.

Please refer to [www.npmjs.com/advisories?search=speedy-ts-compiler](http://www.npmjs.com/advisories?search=speedy-ts-compiler) for more information.

https://www.npmjs.com/package/speedy-ts-compiler

**Comment by cspotcode** [comment, marked as answer]
Oof, thanks for letting me know.

I've updated the docs to refer to a scoped package under my name so that I can guarantee no one else will publish it.

https://typestrong.org/ts-node/docs/transpilers/
